src/operators/video_fade.ml

```diff
@@ -216,7 +222,7 @@ let rec transition_of_string p transition =
             "slide_up";
             "slide_down";
             "fade";
-            "grow";
+            (* "grow"; *)
```

Comment:
Same, can you drop a ticket to restore this one?

Reply:
#1066, should be the place too.

Reply:
Actually just fixed it.